### ISSUE TITLE
🐛 LoO placeholder should uncharm immediately upon being charmed

### ISSUE THREAD

**Issue by TracentEden** [DerpyProjectGroup/topaz#3130]
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my issue will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated.

## Branch affected by issue 

`base`

## Steps to reproduce
- Go to LoO PH and charm the PH and wait

## Expected behavior
- The PH should uncharm immediately and aggro (see https://ffxiclopedia.fandom.com/wiki/Lord_of_Onzozo)
- This might also impact other mobs but will have to check retail as no wiki info (maybe knight crab?)

**Comment by TeoTwawki** [comment]
this is true of most (all?) placeholder mobs on retail.

**Comment by TracentEden** [comment]
I have been able to validate for the LoO PH on retail, however I was not able to validate with, for example, LL placeholder on retail (the PH just stays charmed).

![image](https://user-images.githubusercontent.com/92269743/201500063-5e3abc17-7d78-4e3c-853b-5356004517cc.png)

**Comment by TeoTwawki** [comment]
> LL placeholder on retail (the PH just stays charmed).

Might have to do with their being 2 PHs for 2 LLs, so they didn't think it needed to be protected from holding (the mechanic initially came about due to complains ppl were keeping the PH charmed all day)

I don't know if they have a rule, or it was done case-by case.

**Comment by TracentEden** [comment]
Yeah, seems to be case-by-case basis, VE placeholder is also charmable. Let me know of any others that would be helpful to check.

**Comment by TeoTwawki** [comment]
so much for "most". I'm batting zero lately :sweat_smile:
